runtime/src/lib.rs

```diff
@@ -229,9 +234,9 @@ impl pallet_ethereum_chain_id::Config for Runtime {}
 
 /// Current approximation of the gas/s consumption considering
 /// EVM execution over compiled WASM (on 4.4Ghz CPU).
-/// Given the 500ms Weight, from which 75% only are used for transactions,
-/// the total EVM execution gas limit is: GAS_PER_SECOND * 0.500 * 0.75 => 6_000_000.
-pub const GAS_PER_SECOND: u64 = 16_000_000;
+/// Given the 500ms Weight, from which 65% only are used for transactions,
```

Comment:
@notlesh This was correct before.
A block has 75% dedicated to tx.
65% is for the max of a single transaction